product_examples/errors-and-omissions/products/personal-auto/policy/exposures/vehicle/exposure.json

```diff
@@ -0,0 +1,256 @@
+      "title": "Vehicle Value",
+      "type": "number",
+      "width": 3,
+      "decimalPlaces": 0
+    },
+    {
+      "name": "unrepaired_damage",
+      "title": "Unrepaired Damage",
+      "width": 3,
+      "optional": true,
+      "type": "select",
+      "values": [
+        "No - Existing Damage",
+        "Yes - Existing Damage"
+      ]
+    },
+    {
+      "name": "ownership_indicator",
+      "title": "Ownership Indicator",
+      "width": 3,
+      "optional": true,
+      "type": "select",
+      "values": [
+        "Owned - Not Financed",
+        "Owned - Financed",
+        "Leased"
+      ]
+    },
+    {
+      "name": "vehicle_plated_state",
+      "title": "Licensed State",
+      "newline": true,
+      "type": "select",
+      "width": 3,
+      "optional": true,
+      "values": [
+        "AL",
+        "AK",
+        "AZ",
+        "AR",
+        "CA",
+        "CO",
+        "CT",
+        "DE",
+        "FL",
+        "GA",
+        "HI",
+        "ID",
+        "IL",
+        "IN",
+        "IA",
+        "KS",
+        "KY",
+        "LA",
+        "ME",
+        "MD",
+        "MA",
+        "MI",
+        "MN",
+        "MS",
+        "MO",
+        "MT",
+        "NE",
+        "NV",
+        "NH",
+        "NJ",
+        "NY",
+        "NC",
+        "ND",
+        "OH",
+        "OK",
+        "OR",
+        "PA",
+        "RI",
+        "SC",
+        "SD",
+        "TN",
+        "TX",
+        "UT",
+        "VT",
+        "VA",
+        "WA",
+        "WV",
+        "WI",
+        "WY"
+      ]
+    },
+    {
+      "name": "license_plate",
+      "title": "License Plate Number",
+      "type": "string",
+      "width": 3,
+      "optional": true
+    },
+    {
+      "newline": true,
+      "name": "annual_miles",
+      "title": "Annual Miles",
+      "width": 3,
+      "type": "number",
+      "optional": true
+    },
+    {
+      "name": "primary_vehicle_use",
+      "title": "Primary Vehicle Use",
+      "width": 3,
+      "optional": true,
+      "type": "select",
+      "values": [
+        "Pleasure Only",
+        "Drive to Work / School",
+        "Business - Sales - Artisan",
+        "Farm"
+      ]
+    },
+    {
+      "name": "travel_radius",
+      "title": "Travel Radius",
+      "width": 3,
+      "type": "select",
+      "values": [
+        "Less than 10 Miles",
+        "Between 11 to 20 Miles",
+        "Over 20 Miles"
+      ],
+      "condition": {
+        "primary_vehicle_use": [
+          "Business - Sales - Artisan"
+        ]
+      }
+    },
+    {
+      "name": "travel_radius_farm",
+      "title": "Radius",
+      "width": 3,
+      "type": "select",
+      "values": [
+        "Within 150 miles of Farm",
+        "Intra State",
+        "Multiple States"
+      ],
+      "condition": {
+        "primary_vehicle_use": [
+          "Farm"
+        ]
+      }
+    },
+    {
+      "name": "distance_to_work_school",
+      "title": "Distance One Way",
```

Comment:
It might make it easier to interpret/modify this LOB if the three radius options here follow the same naming convention. Something like "radius_sales", "radius_farm" and "radius_bus". It would be easier for someone viewing this code for the first time to interpret things.